hardhat.config.ts

```diff
@@ -0,0 +1,118 @@
+import * as dotenv from "dotenv";
+import { task } from "hardhat/config";
+import { HardhatUserConfig } from 'hardhat/types';
+import 'hardhat-deploy';
+import "hardhat-gas-reporter";
+import "solidity-coverage";
+import "@nomiclabs/hardhat-ethers";
+import "hardhat-erc1820";
+
+import {
+  RSK_TEST_NET_CHAIN_ID,
+  RSK_MAIN_NET_CHAIN_ID,
+  KOVAN_TEST_NET_CHAIN_ID
```

Comment:
we are not going to deploy this in Kovan

Reply:
done at https://github.com/rsksmart/swap-rbtc/pull/1/commits/f99d281695b2c0d76f0de010564d8a1952ed7b53